helm/kong-app/Chart.yaml

```diff
@@ -1,15 +1,18 @@
 apiVersion: v1
 description: The Cloud-Native Ingress and API-management
 engine: gotpl
-home: https://konghq.com/
-icon: https://s.giantswarm.io/app-icons/1/png/kong-app-light.png
+home: https://github.com/giantswarm/kong-app
+icon: https://s.giantswarm.io/app-icons/kong/1/light.svg
 maintainers:
 - name: hbagdi
   email: harry@konghq.com
 - name: rainest
   email: traines@konghq.com
 name: kong-app
 sources:
-- https://github.com/giantswarm/kong-app/blob/master/README.md
+- https://konghq.com/
 version: 2.2.0
```

Comment:
```suggestion
version: 2.3.2
```

Reply:
Suggestion from @webwurst is correct. The `version` field is overwritten in CI by app-build-suite. The last one crafting a release forgot to update. For now this is manually. Unsolved ticket automating update is https://github.com/giantswarm/giantswarm/issues/16775